### ISSUE TITLE
"Abandon Transaction" doesn't work

### ISSUE THREAD

**Issue by wchancao** [cryptoadvance/specter-desktop#1491]
A couple of days ago i received some sats that never were confirmed, the tx is not in my mempool, looked for it in several block explorers and is not there neither, i click on "Abandon Transaction" on Specter but is still showing that unconfirmed transaction and doesn't delete it from History.

Specter v1.7.1

**Comment by k9ert** [comment]
Your're talking about the button shown here, right: #991

I've never used that button myself. Can you check the logs when you click the button?
CC @kdmukai

**Comment by wchancao** [comment]
Yes, that button... This is what logs shows when i click the button.
![specter_error](https://user-images.githubusercontent.com/488773/144148616-1c1b9775-e26b-4ae7-82df-ff792a6f349d.png)

**Comment by kdmukai** [comment]
I'm not sure. The only issue I can think of is that the abandon tx tests were all around an outgoing tx. I don't know if there'd be any reason for an incoming tx to be handled differently.

**Comment by k9ert** [comment]
Is that transaction still visible on your side? I would assume that it's not because i don't know where it could be persisted.

**Comment by wchancao** [comment]
Yep, it is still there on my tx list...

**Comment by relativisticelectron** [comment]
I get this error too. (mainnet and regtest)

**Comment by relativisticelectron** [comment]
The rpc output should contain the field `"abandoned" : true|false  `  in `gettransaction`   (see: https://github.com/bitcoin-dot-org/developer.bitcoin.org/blob/master/reference/rpc/gettransaction.rst) and in [listtransactions](https://github.com/bitcoin-dot-org/developer.bitcoin.org/blob/master/reference/rpc/listtransactions.rst)
but I get
![image](https://user-images.githubusercontent.com/60378539/168309497-d2115ce3-67ba-442f-8816-c9d443e1f5e4.png)


This makes it impossible to detect here https://github.com/cryptoadvance/specter-desktop/blob/767d5a88b80454f391fd9b31cae348373a5dc607/src/cryptoadvance/specter/wallet.py#L387  if a transaction is abandoned.
